services/ui-src/src/components/icons/Icon.tsx

```diff
@@ -4,36 +4,54 @@ import { IconType } from "../../utils/types/types";
 import { Icon as ChakraIcon } from "@chakra-ui/react";
 // assets
 import {
-  BsBoxArrowRight,
   BsArrowRightSquare,
+  BsBoxArrowRight,
   BsChevronDown,
+  BsFileEarmarkSpreadsheetFill,
   BsPencilSquare,
   BsPersonCircle,
+  BsPlus,
   BsQuestionCircleFill,
 } from "react-icons/bs";
+import { FiMinus } from "react-icons/fi";
+import { HiDownload } from "react-icons/hi";
 
 const iconMap: { [char: string]: IconType } = {
   arrowRightSquare: BsArrowRightSquare,
   boxArrowRight: BsBoxArrowRight,
   chevronDown: BsChevronDown,
+  downloadArrow: HiDownload,
+  spreadsheet: BsFileEarmarkSpreadsheetFill,
   pencilSquare: BsPencilSquare,
   personCircle: BsPersonCircle,
+  plus: BsPlus,
   questionCircleFill: BsQuestionCircleFill,
+  minus: FiMinus,
 };
 
-export const Icon = ({ icon }: Props) => {
-  return <ChakraIcon as={iconMap[icon]} sx={sx.icon} />;
+export const Icon = ({ icon, boxSize, color, fontSize, margin }: Props) => {
+  return (
+    <ChakraIcon
+      as={iconMap[icon]}
+      sx={sx.icon}
+      boxSize={boxSize}
+      margin={margin}
+      fontSize={fontSize}
+      color={color}
```

Comment:
We might be getting enough at this point that we could just do like an `sxOverride` prop that would allow us to pass in a style object.